core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -25,34 +24,50 @@
 
```

Comment:
Could you link to the page with that information?

From my understanding, FilePlayMain (our extended ApplicationListener) delegates render calls to the Play Screen's render method, which delegates to the TriviaButtonBuilder's render method.  If this class isn't thread safe, simply making it inherit from Game won't make it thread safe.